### DISCUSSION TITLE
forgets settings

### DISCUSSION THREAD

**Discussion by wd401959** [File-New-Project/EarTrumpet#645] [Ideas]
Can you please create an ini file to remember settings for various apps volume settings

**Comment by tobbi007** [comment]
would really like to see this happening some time.

**Comment by roncli** [comment]
Agreed, there are times where EarTrumpet just randomly seems to set things to ridiculously low volumes.  Tends to happen when using voice applications that attenuate other application volumes.

**Comment by riverar** [comment]
EarTrumpet just surfaces Windows audio sessions and their internals. But will consider this, thanks.

**Comment by ams2990** [comment]
+1, I want to adjust an app's volume once and have it persist.

**Comment by wjwaldron** [comment]
Yes, definitely!

**Comment by equin0x80** [comment]
It seems to me that the volume levels are preserved correctly for traditional desktop applications but not for the new-style "universal apps" in Windows 10. For example, every time I launch Microsoft Mahjong (from the Windows Store), the volume bar is at 100%, and the sounds are annoyingly loud.

**Comment by Wuestengecko** [comment]
What @equin0x80 said is in fact a design decision. They actually completely removed volume mixing for metro-style (aka Modern UI) apps in Win8, to "make the user experience easier" (i guess). See the following blog post: https://blogs.msdn.microsoft.com/windowsappdev/2012/08/23/media-playback-what-you-need-to-know-about-playing-media-to-make-your-app-shine-in-windows-8/

It basically boils down to "it might confuse some users, and also we didn't know how to neatly integrate it into Windows 8, so we entirely removed it". ET being able to mix them at all might be because it ignores some info provided by the backend that removes the metro apps from the stock mixer. (Please keep ignoring this info, if that's the case! ;) )

**Comment by roncli** [comment]
I do not believe this to be related to Universal Apps.  The app I am using that attenuates other apps' sounds is Discord, and afterwards I often see Chrome and System Sounds set extremely low after use.

To be fair, it's an extremely rare bug, and I can't consistently reproduce it, but it does happen occasionally.

**Comment by riverar** [comment]
Can't see how this is related to EarTrumpet. Do you have audio ducking enabled? It can typically kick in at unwanted times, despite what the dialog says.

![image](https://cloud.githubusercontent.com/assets/475132/17960481/27f44276-6a5b-11e6-822d-384c95a13a56.png)

**Comment by roncli** [comment]
My Communications tab matches yours, "Do nothing" is the selected option.

This is the feature in Discord that attenuates (I think that's also audio ducking).

![image](https://cloud.githubusercontent.com/assets/498929/17960548/95b4a224-6a5b-11e6-8b34-8f06614e7719.png)
